near/Cargo.toml

```diff
@@ -21,6 +21,8 @@ serde_json = "1.0"
 lazy_static = "1.4"
 dirs = "2.0.2"
 borsh = "0.2.10"
+tracing = "0.1"
```

Comment:
Shouldn't this be behind a feature flag?

Reply:
I don't believe this is possible. It is like logging, i.e. the macroses are used anyway, and the tracing level is checked at runtime (`tracing` crate even supports updating the logging level on the fly: https://youtu.be/JjItsfqFIdo?t=495)

Reply:
@nearmax no, the idea is to ultimately replace `log` with `tracing` since the latter integrate very nicely with async programs, and transition can be done smoothly. In this PR `log`'s subscribers were already replaced by `tracing`'s subscribers.

It offer several advantages that I'll try to summarize, maybe it is a good fit for an All Hands Presentation.